drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
+#include <unordered_map>
```

Comment:
Standard library is [not permitted](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#standard-template-library)

Reply:
You should use `HashMap` instead

Reply:
Switched to HashMap.